docs/openlaw-object/README.md

```diff
@@ -0,0 +1,2141 @@
+# Openlaw Object
+
+The `Openlaw` object defined in Openlaw.scala compiles to JavaScript and is an interface in the OpenLaw protocol to interact directly with an agreement and its contents, including its various variable types. The object methods are categorized below.
+
+## Template
+
+### compileTemplate
+
+Compile a template's content including markup language.
+
+```scala
+compileTemplate(
+  text: String
+): js.Dictionary[Any]
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `text` | `String` | **Required.** The raw content of the template (including markup language) to be compiled. |
+
+Example
+
+```js
+Openlaw.compileTemplate("This Advisor Agreement is entered into between [[Company Name]] (\"Corporation\") and [[Advisor Name]] (\"Advisor\") as of [[Effective Date: Date]] (\"Effective Date\"). Company and Advisor agree as follows: \n\n^**Services**. Advisor agrees to consult with and advise Company from time to time, at Company's request (the \"Services\").");
+```
+
+**Response**
+
+Returns an object containing a [`CompiledTemplate` object](#compiledtemplate) (and key/value pairs indicating that no error has occurred) if compilation is successful.
+
+Example
+
+```js
+{
+  isError: false,
+  errorMessage: "",
+  compiledTemplate: CompiledTemplate
+}
+```
+
+### execute
+
+Execute (load and run) a compiled template.
```

Comment:
Should we explain a bit more what it means to load and run a template or are we assuming that information will be familiar to someone who is reading?

Reply:
@adridadou what are your thoughts on this one? Is there something more we can add here? I included "load and run" to describe "execute" to distinguish from the legal meaning of execute (all the parties signing the contract).